src/@components/common/GrayGap.tsx

```diff
@@ -0,0 +1,16 @@
+import { styled } from "styled-components";
+
+export default function GrayGap() {
+  return (
+    <>
+      <GreyGap></GreyGap>
```

Comment:
요고는 뭔가 children부분이 채워지는 부분이 아니니까, 
```suggestion
      <GreyGap/>
```
이렇게 써주어보는 것은 어떨까용?